static/src/javascripts/projects/commercial/modules/third-party-tags/facebook-pixel.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import config from 'lib/config';
+import {
+    getAdConsentState,
+    thirdPartyTrackingAdConsent,
+} from 'common/modules/commercial/ad-prefs.lib';
+
+export const fbPixel: () => ThirdPartyTag = () => {
+    const consent = getAdConsentState(thirdPartyTrackingAdConsent);
+    return {
+        shouldRun:
+            config.switches.facebookTrackingPixel &&
+            (consent == null || consent),
```

Comment:
Can consent be `undefined`? Are we intentionally including it in the check?

```js
undefined == null       // true
undefined === null       // false
```

Reply:
Ah, just answered my own Q by looking at the codebase:

```js
const getAdConsentState = (provider: AdConsent): boolean | null => {
```

so it will either be `boolean` or `null` 😁 Nevermind me!

Reply:
I'm going on the type annotation found here and covering those cases.

https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/commercial/ad-prefs.lib.js#L31